doc/dev/adr/0023-property-support.md

```diff
@@ -0,0 +1,312 @@
+
+* The request payload is an instance of `TCtrl`.
+  * For statically itemized properties, the optional fields in the `TCtrl` request object are `true` for each field whose Property is to be read.
+  * For dynamically itemized properties, the elements in the `TCtrl` request array indicate the names of the Properties to read.
+* The response payload is an instance of `TProp`.
+  * For statically itemized properties, the optional fields in the `TProp` response object contain values for any Properties that have been read.
+  * For dynamically itemized properties, the keys in the `TProp` response map indicate which Properties have been read, and the corresponding values are what was read.
+
+The *observe* action is performed by issuing an 'observe' Command request.
+
+* The request payload is an instance of `TCtrl`.
+  * For statically itemized properties, the optional fields in the `TCtrl` request object are `true` for each field whose Property is to be added to the notify list.
+  * For dynamically itemized properties, the elements in the `TCtrl` request array indicate the names of the Properties to add to the notify list.
+  * The maintainer may keep a single notify list across all consumers, or it may keep a separate notify list for each consumer; in the latter case, the maintainer must be provided with a way to identify the consumer that requested the observation.
+* The response payload is an instance of `TStat`.
+  * For statically itemized properties, each optional field in the `TStat` response object indicates the status of the observe operation for the Property named by the field.
+  * For dynamically itemized properties, each value in the `TStat` response map indicates the status of the observe operation for the Property named by the key.
+  * The semantics of `TStatVal` values are determined by the service that implements the property.
+  * The maintainer's response may additionally provide status values for the observation state of properties other than those named in the 'observe' request.
+
+The *unobserve* action is performed by issuing an 'unobserve' Command request.
+
+* The request payload is an instance of `TCtrl`.
+  * For statically itemized properties, the optional fields in the `TCtrl` request object are `true` for each field whose Property is to be removed from the notify list.
+  * For dynamically itemized properties, the elements in the `TCtrl` request array indicate the names of the Properties to remove from the notify list.
+  * The maintainer may keep a single notify list across all consumers, or it may keep a separate notify list for each consumer; in the latter case, the maintainer must be provided with a way to identify the consumer that requested the observation.
+* The response payload is an instance of `TStat`.
+  * For statically itemized properties, each optional field in the `TStat` response object indicates the status of the unobserve operation for the Property named by the field.
```

Comment:
would this describe everything that was successfully unobserved, or would this response describe the state of the current (still active) observations? I think there are clarity arguments for either option, unfortunately

Reply:
Given the flexibility that is currently proposed regarding status values, I think the application is free to imply any choice of default value to non-present values.